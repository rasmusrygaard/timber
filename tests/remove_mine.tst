remove /
remove //
remove ///
remove /foo
remove /foo!
create /a 1
create /a/a 1
create /a/a/a 1
create /a/b 1
remove /a
remove /a/a
remove /a/a/
remove /a/b
remove /a/a/a
remove /a
remove /a/a
remove /a
remove /xxx
