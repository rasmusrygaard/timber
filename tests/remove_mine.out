MALFORMED KEY
MALFORMED KEY
MALFORMED KEY
KEY NOT FOUND
MALFORMED KEY
CREATED
CREATED
CREATED
CREATED
HAS CHILDREN
HAS CHILDREN
MALFORMED KEY
REMOVED
REMOVED
HAS CHILDREN
REMOVED
REMOVED
KEY NOT FOUND
